test/common/http/codec_impl_fuzz_test.cc

```diff
@@ -230,31 +244,46 @@ class ReorderBuffer {
 
   bool empty() const { return bufs_.empty(); }
 
-  Http2::ConnectionImpl& connection_;
+  Connection& connection_;
   std::deque<Buffer::OwnedImpl> bufs_;
 };
 
 typedef std::unique_ptr<HttpStream> HttpStreamPtr;
 
-// Fuzz the H2 codec implementation.
-DEFINE_PROTO_FUZZER(const test::common::http::CodecImplFuzzTestCase& input) {
+namespace {
+
+void codecFuzz(const test::common::http::CodecImplFuzzTestCase& input, bool http2) {
```

Comment:
nit: prefer 2 value enum over bool param for ease of reading

Reply:
Ack; I will clean this up in a follow PR with additional corpus entries, want to throw this into the overnight ClusterFuzz run so will merge for now.